### PR TITLE
python3Packages.pymetno: 0.8.2 -> 0.8.3

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 0.8.3

Change log: https://github.com/Danielhiversen/pyMetno/releases/tag/0.8.3

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120465` at 0bab1756 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.pymetno</li>
    <li>python39Packages.pymetno</li>
  </ul>
</details>

File: pkgs/development/python-modules/pymetno/default.nix [hunk 1]
```diff
@@ -8,15 +8,15 @@
 }:
 
 buildPythonPackage rec {
-  pname = "PyMetno";
-  version = "0.8.2";
+  pname = "pymetno";
```
Comment (mweinelt) [inline pkgs/development/python-modules/pymetno/default.nix:11]: https://github.com/Danielhiversen/pyMetno/blob/master/setup.py#L4

Why the change?

Comment (SuperSandro2000) [reply pkgs/development/python-modules/pymetno/default.nix:11]: pname should match the name in python-packages.

Comment (fabaff) [reply pkgs/development/python-modules/pymetno/default.nix:11]: The [docs](https://nixos.org/manual/nixpkgs/unstable/#contributing) says we follow [PEP-0503]https://www.python.org/dev/peps/pep-0503/#normalized-names) for the names.
